fab/fabfile.py

```diff
@@ -489,38 +494,35 @@ def development():
     env.django_port = '9010'
     env.should_migrate = True
 
-    require('inventory')
-
-    # use inventory filename as environment name
-    # i.e. if the inventory is called my-crazy-setup
-    # then things on the server will be stored in
-    # /home/cchq/www/my-crazy-setup/code_root, etc.
-    env.environment = os.path.basename(env.inventory)
+    require('inventory', 'environment')
```

Comment:
Is it safe to assume environment=dev if someone has typed `fab development ...`?